src/js/components/CopyToClipboard.js

```diff
@@ -26,20 +26,23 @@ export default class extends React.PureComponent {
     }
 
     handleCopy = () => {
-        const container = document.createElement('textarea');
         const { clickCallback, src, namespace } = this.props;
 
-        container.innerHTML = JSON.stringify(
+        const textToCopy = JSON.stringify(
             this.clipboardValue(src),
             null,
             '  '
         );
 
-        document.body.appendChild(container);
-        container.select();
-        document.execCommand('copy');
-
-        document.body.removeChild(container);
+        if (navigator.clipboard) {
```

Comment:
Note: This will cause this to not work in non-secure context with no way for the developer to recover (other than using a polyfil), this may be worth mentioning in the docs.

Reply:
true. it will work with `https` and `localhost`, but not with `http`. we could keep a fallback:

```js
            navigator.clipboard.writeText(textToCopy).catch(err => {
                // Fallback for non-secure contexts (i.e. http)
                const textArea = document.createElement('textarea');
                textArea.value = textToCopy;
                document.body.appendChild(textArea);
                textArea.select();
                document.execCommand('copy');
                document.body.removeChild(textArea);
            });
```

@Kikobeats?

Reply:
LGTM 🙂